jac-uk/admin
# #1723 Diversity Report - Amendments

## What's included?
For the diversity reports:

- candidates listing their ethnicity as Other should be included in the BAME category
- display a declaration rate (see example below)
- candidates who have Not Answered or have selected Other or Prefer Not To Say should not be included in the percentage calculations, eg out of 10 cats:

• 6 said they prefer Whiskers
• 2 said they prefer Applause
• 2 preferred not to say

The percent would be:

• 75% said they prefer Whiskers (6 out of 8)
• 25% said they prefer Applause (2 out of 8)
• Declaration rate was 80% (8 out of 10)

This is linked to the following PRs in Digital Platform:

https://github.com/jac-uk/digital-platform/pull/904
https://github.com/jac-uk/digital-platform/pull/933

And the following PR in Apply: https://github.com/jac-uk/apply/pull/1049

Closes #1723 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
**Please use this Production url to view the diversity report for live exercises:**
https://jac-apply-admin-production--1723-diversity-y27mp62o.web.app/

Go to the diversity report and ensure that the numbers look right according to the criteria above.

Declaration Total has been removed for the following:
- Social Mobility
- EMP Status

Ensure that the Dashboard page reflects the same figures.

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

---
PREVIEW:PRODUCTION
_can be OFF, DEVELOP or STAGING_
